back-end/src/test/java/admin/job/application/JobServiceTest.java

```diff
@@ -141,6 +140,22 @@ void findByNotExistingId() {
                 .isEqualTo(JobException.JOB_NOT_FOUND.getException());
     }
 
+    private Long saveGpuServerInLab(Lab lab) {
+        GpuServer server = new GpuServer("server", true, 1024L, 1024L, lab);
+        gpuServerRepository.save(server);
+
+        GpuBoard board = new GpuBoard(false, 600L, "nvdia", server);
+        gpuBoardRepository.save(board);
+
+        return server.getId();
+    }
+
+    private Long saveMember(Lab lab) {
+        Member member = new Member("email", "password", "name", MemberType.USER, lab);
```

Comment:
```suggestion
        Member member = new Member(USER_EMAIL, PASSWORD, NAME, MemberType.USER, lab);
```
MemberFixture 의 스태틱 상수를 public으로 열어준 것 같아서 위와 같이 제안해 봅니다. 😀

Reply:
대부분의 방식에서 수정이 필요한 부분이라 우선은 지금 상태를 유지하겠습니다!